scripts/nodesource-node22.sources

```diff
@@ -0,0 +1,34 @@
+Types: deb
+URIs: https://deb.nodesource.com/node_22.x
```

Comment:
nodesource is the primary source of updated debs for different nodejs versions, because Debian tends to lag behind a lot. It's also what Wikimedia uses.